incubator/binding-asyncapi/src/test/java/io/aklivity/zilla/runtime/binding/asyncapi/internal/stream/client/AsyncapiIT.java

```diff
@@ -69,8 +69,8 @@ public void shouldPublishAndSubscribe() throws Exception
     @Test
     @Configuration("client.http.yaml")
     @Specification({
-        "${asyncapi}/create.item/client",
-        "${http}/create.item/server"
+        "${asyncapi}/http/create.pets/client",
+        "${http}/create.pets/server"
```

Comment:
```suggestion
        "${asyncapi}/http/create.pet/client",
        "${http}/create.pet/server"
```

Also test method name below should be `shouldCreatePet`.